src/main/scala/ai/starlake/job/ingest/IngestionJob.scala

```diff
@@ -752,26 +752,32 @@ trait IngestionJob extends SparkJob {
         tags = schema.tags,
         writeStrategy = Some(strategy)
       )
-      val autoTask = new SparkAutoTask(taskDesc, Map.empty, None, truncate = false, test = test)(
-        settings,
-        storageHandler,
-        schemaHandler
-      )
-      if (autoTask.sink(mergedDF)) {
+      val autoTask =
         taskDesc.getSinkConfig() match {
-          case fsSink: FsSink =>
-            if (isCSV(fsSink)) {
-              val separator = mergedMetadata.separator
-              val header =
-                if (mergedMetadata.resolveWithHeader())
-                  Some(mergedDF.columns.toList)
-                else None
-              autoTask.exportToCSV(domain.finalName, schema.finalName, header, separator)
-            }
+          case fsSink: FsSink if isCSV(fsSink) =>
```

Comment:
The behavior is different from before, is it the behavior you want ?

Previous behavior was to ignore export if it wasn't CSV sink but now we fallback to spark auto task if it's not a csv file sink.